plugins/balloontoolbar/plugin.js

```diff
@@ -665,9 +675,7 @@
 				this._detachListeners();
 
 				function attachListener() {
-					this.attach( this._pointedElement, {
-						focusElement: false
-					} );
+					this.reposition( { focusElement: false } );
```

Comment:
You pass parameter to the `reposition` method here, yet it does not accept any parameters. IMO it should, as it should pass options to the `attach`.

Reply:
I wonder if reposition should really accept options, as `balloonToolbar.attach` doesn't.